builder/builder_test.go

```diff
@@ -508,3 +486,16 @@ func checkDepositTxResult(t *testing.T, txStore txstore.TxStore, depositTx bftty
 	require.Equal(t, cosmAddr, actualReceiver, "Receiver address mismatch")
 	require.Equal(t, expectedDepositAmount, actualDepositAmount, "Deposit amount mismatch")
 }
+
+func checkEventBusEventAndGetData(t *testing.T, subscription bfttypes.Subscription) bfttypes.EventDataTx {
```

Comment:
nit: could you rename this to either `getEventData` or `getEventDataTx`?

Reply:
Resolved